src/main/webapp/i18n/de/course.json

```diff
@@ -42,9 +42,21 @@
             "fileUploadExercises": "Datei Upload Aufgabe",
             "students": "Studenten",
             "tutors": "Tutoren",
-            "instructors": "Instructoren",
+            "instructors": "Instruktoren",
             "onlineCourseFalse": "Nein",
-            "onlineCourseTrue": "Ja"
+            "onlineCourseTrue": "Ja",
+            "form": {
```

Comment:
I saw that there was a translation in the `en` language file, but is missing in the `de` language file, so I updated it